ee/clickhouse/queries/funnels/test/test_funnel.py

```diff
@@ -132,44 +124,6 @@ def test_advanced_funnel_with_repeat_steps(self):
         self.assertEqual(result[1]["name"], "$pageview")
         self.assertEqual(result[4]["name"], "$pageview")
         self.assertEqual(result[0]["count"], 5)
-        # check ordering of people in every step
-        self.assertCountEqual(
```

Comment:
*memorizes commit to rewrite test for funnel/persons*

Instead of just removing the people check, can we please still have the "count" check for each step? So we know the funnel is still working as expected?

Right now, these tests reduce my confidence in funnels implementation quite a bit.

Reply:
Coming back to this after seeing https://github.com/PostHog/posthog/pull/4883 - can we move all of this to that PR?

I'm thinking: test both these APIs together (I think that's how they'll be used anyway, isn't it? And it makes debugging tests easier, when you know exactly why user got borked)

So these tests can be revamped in https://github.com/PostHog/posthog/pull/4883 to call both classes, the first class to test the counts, the second class to test the people?

Reply:
I think that's fair idea but I don't like the concept of having the functionalities rely on each other in tests. A broken person api could make you think the actual step counts are wrong but you won't be sure since these two tests are wrapped together

Reply:
yeah i'm down for that. It would be a sort of integration test. Could we do this addition right after these two PRs are merged? Since we already have them in place, I'd rather get them in so core experience can start using them

Reply:
Yes! :)